rest-api-spec/src/main/resources/rest-api-spec/api/cluster.allocation_explain.json

```diff
@@ -10,6 +10,10 @@
         "include_yes_decisions": {
           "type": "boolean",
           "description": "Return 'YES' decisions in explanation (default: false)"
+        },
+        "wait_for_active_shards": {
```

Comment:
If bulk has it then reindex, update-by-query, and delete-by-query should have it. They copied `consistency` to their bulk requests so they should copy this instead.